usr/sbin/registercloudguest

```diff
@@ -351,7 +353,9 @@ def setup_ltss_registration(registration_target, regcode):
         suseconnect = run_SUSEConnect(
             registration_target=registration_target,
             product=ltss_product_triplet,
-            regcode=regcode
+            regcode=regcode,
+            instance_data_filepath=instance_data_filepath
+            if os.path.exists(instance_data_filepath) else ''
```

Comment:
This already happens in the called function, no need to to the same operation twice. Fixed, force pushed.